doc/source/devel/make_release.rst

```diff
@@ -130,11 +130,9 @@ Release checklist
 * Check on different platforms, particularly windows and PPC. Look at the
   `nipy buildbot`_ automated test runs for this.
 
-* Check the documentation doctests::
+* Check the documentation doctests (forcing Python 2)::
```

Comment:
That's true. I'll change it to `python2`, because sphinx is not Python 3 safe, yet.

I'll [`six`-ify](https://github.com/effigies/nibabel/tree/sphinx3) it as I have time.